cmd/crc/cmd/config/config.go

```diff
@@ -11,20 +11,20 @@ import (
 )
 
 const (
-	Bundle               = "bundle"
-	CPUs                 = "cpus"
-	Memory               = "memory"
-	DiskSize             = "disk-size"
-	NameServer           = "nameserver"
-	PullSecretFile       = "pull-secret-file"
-	DisableUpdateCheck   = "disable-update-check"
-	ExperimentalFeatures = "enable-experimental-features"
-	NetworkMode          = "network-mode"
-	HTTPProxy            = "http-proxy"
-	HTTPSProxy           = "https-proxy"
-	NoProxy              = "no-proxy"
-	ProxyCAFile          = "proxy-ca-file"
-	EnableMonitoring     = "enable-monitoring"
+	Bundle                    = "bundle"
+	CPUs                      = "cpus"
+	Memory                    = "memory"
+	DiskSize                  = "disk-size"
+	NameServer                = "nameserver"
+	PullSecretFile            = "pull-secret-file"
+	DisableUpdateCheck        = "disable-update-check"
+	ExperimentalFeatures      = "enable-experimental-features"
+	NetworkMode               = "network-mode"
+	HTTPProxy                 = "http-proxy"
+	HTTPSProxy                = "https-proxy"
+	NoProxy                   = "no-proxy"
+	ProxyCAFile               = "proxy-ca-file"
+	EnableOpenShiftMonitoring = "enable-openshift-monitoring"
```

Comment:
Couldn't this get confusing on okd where we should avoid talking about openshift?

Reply:
Good point! What about `enable-cluster-monitoring-operator` ? At least this is exactly what it does.

Reply:
Renamed to enable-cluster-monitoring.